dev-lang/nasm/nasm-2.15.05.ebuild

```diff
@@ -1,9 +1,7 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
1. If RDEPEND/DEPEND are empty, you can drop them.
2. S= needs quotes; and please move it closer to SRC_URI
3. Perhaps install needs ${ED} instead of ${D}?

Reply:
Thanks for the suggestions!

I should probably write a pre-commit hook to check the order of everything because I always seem to forget...